### DISCUSSION TITLE
How to disable Hide Entry Points for Fast User Switching in the new version of the script.

### DISCUSSION THREAD

**Discussion by ivaluexrays** [HotCakeX/Harden-Windows-Security#214] [Q&A]
The option is now included by default in the Microsoft Security Baselines. However, I was unable to disable this specific option through GPOs and am currently exploring using the registry for this. Could I potentially receive some advice for how to disable this specific feature? Any help would be appreciated!

**Comment by HotCakeX** [comment]
Hi,
It's not included in Microsoft security baselines, it's always been in the [User Account Control category](https://github.com/HotCakeX/Harden-Windows-Security?tab=readme-ov-file#user-account-control). If it was included in MS security baselines I wouldn't put it on a separate policy in the script like this, to prevent duplication 🙂

![image](https://github.com/HotCakeX/Harden-Windows-Security/assets/118815227/416008fb-9468-4d0a-808e-9194cc9acbe8)

<br>

To disable it, you need to use group policy again, otherwise after some time or a restart it will revert back, because group policies are made in a way to keep the system compliant with the policies.

To disable that policy you can set it to "Not configured" state like this

<img width="1280" alt="image" src="https://github.com/HotCakeX/Harden-Windows-Security/assets/118815227/c932550f-4bf7-44b1-8eb0-6dce1939d59d">

<br>

<br>

The [Unprotect-WindowsSecurity](https://github.com/HotCakeX/Harden-Windows-Security/wiki/Harden%E2%80%90Windows%E2%80%90Security%E2%80%90Module) cmdlet removes it as well of course, along with pretty much everything else.

**Reply by ivaluexrays** [reply]
Hi, sorry I might have been looking at the wrong group policy. I was just trying to restore the feature of allowing an admin to enter their PIN/biometrics on the secure desktop for UAC elevation. While I can unblock UAC elevation, I was never able to restore the other feature. I just noticed that it happened after I enabled the Microsoft Security Baselines and am not sure what's causing it now.

**Reply by HotCakeX** [reply]
I think you're looking for this one

<img width="844" alt="image" src="https://github.com/HotCakeX/Harden-Windows-Security/assets/118815227/9040f60e-93f1-421f-859c-dcb23e8ef7d0">

<br>

You can change it here

<img width="1280" alt="image" src="https://github.com/HotCakeX/Harden-Windows-Security/assets/118815227/419fb95e-3e14-40e3-abb0-159a06067146">

<br>

<br>

If it's not what you were referring to then the one above it might be

**Reply by ivaluexrays** [reply]
It does prompt for credentials, but I noticed that before, I had the option of choosing between PIN, biometrics, or Standard Username + Password. I was wondering why that was hidden after running Microsoft Security Baselines
